src/scikit_package/cli/gh.py

```diff
@@ -0,0 +1,16 @@
+def _broadcast_issue_to_urls(issue_content, broadcast_urls):
+    """Broadcast issue to the repos pointed by the urls in the
+    broadcast_urls.
+
+    Parameters
+    ----------
+    issue_content : dict
+        issue_content needed to create issues in each repo.
+    broadcast_urls : list of str
+        urls to the target repos.
+
+    Returns
```

Comment:
maybe return something that tracks success?  Which ones were done successfully and which not.